src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Ball extends ColorSupplier {

```

Comment:
```suggestion
public class Ball {
```
Наследование - это когда один объект является ПО СВОЕЙ СУТИ частным случаем другого (более общего)
Квадрат является Фигурой
Кот является Животным
Мячик является Поставщиком цвета?